torch/csrc/utils/tensor_numpy.cpp

```diff
@@ -125,13 +125,13 @@ PyObject* tensor_to_numpy(const at::Tensor& tensor) {
   return array.release();
 }
 
-at::Tensor tensor_from_numpy(PyObject* obj) {
+at::Tensor tensor_from_numpy(PyObject* obj, bool copy_numpy/*=false*/) {
```

Comment:
This is a historic issue, but since this PR updates this function, anyway, would you add a brief comment explaining what this function does (and what the copy_numpy bool does)?

Reply:
Also: what does `/*=false*/` do in the signature?

Reply:
Final nit on this signature: is `copy_numpy` the correct name for the argument? That would imply that it controls the behavior of the function, but actually this is only used to know whether or not to throw a warning, right?

Would a name like, "warn_if_not_writeable" be more descriptive?

Reply:
> Also: what does `/*=false*/` do in the signature?

It's just a comment to tell the default parameter passed is false.

Reply:
https://github.com/pytorch/pytorch/pull/47271/commits/a4101073298241bacb0d900e7e3a94cf1b4c204a changes the parameter name and adds the comment for reference.